api/src/main/java/io/strimzi/api/kafka/model/KafkaConnectorSpec.java

```diff
@@ -0,0 +1,47 @@
+/*
+ * Copyright 2019, Strimzi authors.
+ * License: Apache License 2.0 (see the file LICENSE or http://apache.org/licenses/LICENSE-2.0.html).
+ */
+package io.strimzi.api.kafka.model;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+import java.net.URL;
+import java.util.List;
+
+public class KafkaConnectorSpec {
+    private ConnectCluster connectCluster;
+    private String className;
+    private Integer tasksMax;
+    private List<String> topics;
+    private List config;
+
+    public List getConfig() {
```

Comment:
There's an `@Description` annotation which you can use on all getters (and on classes too) to provide documentation. It's used by the tooling to generate CRD reference documentation.